CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+
+# Contributing to the BluePhos Pipeline Project
+
+Thank you for your interest in contributing to the BluePhos pipeline project! As a contributor, you’ll work from your fork of the main repository. This document outlines the steps to set up your development environment, guidelines for coding, and instructions for submitting contributions.
+
+## Repository Information
+
+- **Main Repository**: [BluePhos Main Repo](https://github.com/ssec-jhu/bluephos.git)
+
+## Getting Started
+
+1. **Fork the Repository**: First, create a fork of the main repository in your own GitHub account. This will allow you to freely make changes without affecting the main repository.
+
+2. **Clone Your Fork**: Clone your fork to your local machine:
+   ```bash
+   git clone https://github.com/your-username/bluephos.git
+   cd bluephos
+   ```
+
+3. **Add the Main Repository as Upstream**: To keep your fork in sync with the latest updates from the main repo, add it as a second remote named `upstream`:
+   ```bash
+   git remote add upstream https://github.com/ssec-jhu/bluephos.git
+   ```
+
+4. **Create a Branch**: Create a new branch for each feature or bug fix:
+   ```bash
+   git checkout -b feature/your-feature-name
+   ```
+
+## Making Changes
+
+- **Coding Standards**: Follow PEP 8 for Python code. Use `ruff format` or `tox -e format` to ensure code formatting aligns with the repository’s requirements.
```

Comment:
Here I think it would be better to just use the `tox` command. Fewer options is better and tox codifies more than just the formatting - we can change in it's code without changing advice here.

Reply:
That makes sense. Using tox -e format as the single command has clear advantages. I initially used ruff format . because it was mentioned in the CI error, but after running tox -e format, it made additional adjustments on 9 more files :).